CONTRIBUTING.md

```diff
@@ -220,10 +220,16 @@ You can lint a single file (but this will _not_ exclude legacy errors):
 - Style rules are (mostly) defined by `src/uncrustify.cfg` which tries to match
   the [style-guide]. To use the Nvim `gq` command with `uncrustify`:
   ```
-  if !empty(findfile('src/.uncrustify', ';'))
-    setlocal formatprg=uncrustify\ -q\ -c\ src/uncrustify.cfg\ --replace\ --no-backup
+  if !empty(findfile('src/uncrustify.cfg', ';'))
+    setlocal formatprg=uncrustify\ -q\ -l\ C\ -c\ src/uncrustify.cfg\ --no-backup
   endif
   ```
+  If you notice a lot of warnings from `uncrustify` then make sure you are using one with a version number that's higher than the one specified in `src/uncrustify.cfg`. 
```

Comment:
this can be a lot shorter, and don't need the ceremonious `diff ...` script. 

> The required version of uncrustify is specified in uncrustify.cfg.

I'm not even sure this is worth mentioning.